Fix listing a repository without cpeid

jan-kolarik opened the pull request:
Use empty string when constructing a repository object having `cpeid == NULL`. This fixes the exception when using the `repolist` in verbose mode with manually constructed repo.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2066334.

j-mracek commented on libdnf/repo/Repo.cpp:
```diff
@@ -1031,8 +1031,13 @@ bool Repo::Impl::loadCache(bool throwExcept, bool ignoreMissing)
+                if (distroTag->cpeid) {
+                    cpeid_str = distroTag->cpeid;
+                }
+                distro_tags.emplace_back(cpeid_str, distroTag->tag);
```
[Minor] the code will double allocate the memory. It is not an issue but what about to try to avoid it?

j-mracek replied:
Please can you add a reference of bugzilla to the commit message? It helps to identify and link the work with bugzilla.
Sugesting:
```
Fix passing repo with null cpeid (RhBug:2066334)

= changelog =
type: bugfix
resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2066334
```

See: https://github.com/rpm-software-management/libdnf/commit/ef94cfc564b38389abbee6d4da5432e46b49868a

I also recommend to add a reference for the PR in Bugzilla. It allows reporters to participate on testing and it is required for the release to link it with PRs.

Thank you in advance.

jan-kolarik replied:
I hope it is OK now, otherwise please let me know.

jan-kolarik committed:
Fix listing a repository without cpeid (RhBug:2066334)

= changelog =
type: bugfix
resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2066334

j-mracek commented:
LGTM

j-mracek commented:
I will merge PR and tests when all tests pass. Then please move the bug into status - post
